bugwarrior/services/bitbucket.py

```diff
@@ -157,12 +173,13 @@ def validate_config(cls, config, target):
         IssueService.validate_config(config, target)
 
     def fetch_issues(self, tag):
-        response = self.get_data('/repositories/%s/issues/' % (tag))
-        return [(tag, issue) for issue in response['values']]
+        response = self.get_collection('/repositories/%s/issues/' % (tag))
+        for issue in response:
+            yield (tag, issue)
```

Comment:
Why employ a generator here when we just collapse it with a ``list()`` below?

Also, why a generator here and not in ``fetch_pull_requests``.  I don't mind using one over the other, just so long as its consistent.